manifests/s/SillyKoala/IdleEucalyptus/1.1.0/SillyKoala.IdleEucalyptus.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SillyKoala.IdleEucalyptus
+PackageVersion: 1.1.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://gateway.ipfs.io/ipfs/bafybeigd2rpegu5ls6nuzo4fqpxr42peduoe54c5rtrq3s3wnhfpf5iele/idle-eucalyptus-install-1.1.exe

```

Comment:
```
  InstallerUrl: https://gateway.ipfs.io/ipfs/bafybeigd2rpegu5ls6nuzo4fqpxr42peduoe54c5rtrq3s3wnhfpf5iele/idle-eucalyptus-install-1.1.exe
```
Where did this URL come from?

I see a different URL on https://sillykoala.itch.io/idle-eucalyptus